.github/workflows/ci.yaml

```diff
@@ -133,8 +136,10 @@ jobs:
       - name: Update Nix Flake SRI Hash
         run: ./scripts/update-flake.sh
 
-      - name: Ensure No Changes
-        run: git diff --exit-code
+      - uses: stefanzweifel/git-auto-commit-action@v5
+        with:
+          # Allows dependabot to still rebase!
+          commit_message: "[dependabot skip] Update Nix Flake SRI Hash"
```

Comment:
@kylecarbs do we still need this? I think we should if it works. It should make this process automated.